.buildkite/pipeline.yml

```diff
@@ -373,4 +373,15 @@
 - label: ":kubernetes: operator"
   commands:
     - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/travis/upload_build_info.sh; fi }; trap cleanup EXIT
-    - ./ci/travis/prep-k8s-environment.sh
\ No newline at end of file
+    # Set up local kind cluster.
+    - ./ci/travis/prep-k8s-environment.sh
+    # Build py37-cpu Ray image for the test.
+    - LINUX_WHEELS=1 ./ci/travis/ci.sh build
+    - pip install -q docker
+    - python ci/travis/build-docker-images.py --py-versions py37 --device-types cpu --build-type LOCAL --build-base
+    # Tag the image built in the last step. We want to be sure to distinguish the image from the real Ray nightly.
+    - docker tag rayproject/ray:nightly-py37-cpu ray-ci:kuberay-test
+    # Load the image into the kind node.
+    - kind load docker-image ray-ci:kuberay-test
+    # Run the test with the image.
+    - RAY_IMAGE=docker.io/library/ray-ci:kuberay-test pytest -svv --timeout 300 python/ray/tests/kuberay/test_autoscaling_e2e.py
```

Comment:
please bazel-ify this

Reply:
I'll need you're help with that. On my first attempt at "bazel test" I couldn't access K8s from the bazel test sandbox.

Reply:
those should work? we default to local execution (no sandbox) with `--config=ci`. You can also pass in env var with `--test_env`

Reply:
Still getting this when running with Bazel: `The connection to the server localhost:8080 was refused - did you specify the right host or port?`

Will push a commit with my current unsuccessful Bazel attempt.

Reply:
Simon's suggestion of passing KUBECONFIG as a test env works!